src/scss/patterns/_media.scss

```diff
@@ -116,3 +116,27 @@
     justify-self: auto;
   }
 }
+
+// Media Pull
+// ----------
+// Move an image to the left of the content for wide enough screens
+
+.media-pull {
+  --column: page-start;
+
+  @include config.above('xlarge-page') {
+    --column: full-start / page-start;
+
+    grid-row: var(--rowspan);
+    justify-self: end;
+    max-width: 25vw;
+    padding-inline: calc(var(--gutter) + 2vw);
```

Comment:
These numbers feel a bit magical and unclear to me. Why 25vw? Right now, this doesn't align with anything else on the page - should it? Is the extra gutter space meant to avoid our hanging link icons on headings? If you use this next to a paragraph rather than a heading, does the extra space still make sense?